vector/src/main/java/im/vector/preference/BingRulePreference.kt

```diff
@@ -149,11 +155,44 @@ class BingRulePreference : VectorPreference {
         return rule
     }
 
+    override fun onBindViewHolder(holder: PreferenceViewHolder) {
+        super.onBindViewHolder(holder)
+        val radioGroup = holder.findViewById(R.id.bingPreferenceRadioGroup) as? RadioGroup
+
+        when (ruleStatusIndex) {
+            NOTIFICATION_OFF_INDEX -> {
+                radioGroup?.check(R.id.bingPreferenceRadioBingRuleOff)
+            }
+            NOTIFICATION_SILENT_INDEX -> {
+                radioGroup?.check(R.id.bingPreferenceRadioBingRuleSilent)
+            }
+            else -> {
+                radioGroup?.check(R.id.bingPreferenceRadioBingRuleNoisy)
+            }
+        }
+
+        radioGroup?.setOnCheckedChangeListener { group, checkedId ->
```

Comment:
This is maybe the source of issue we have seen this morning. Have you fixed it?

Reply:
Fixed it, listener is set to null before initializing radiogroup value